admin_guide/scheduling/priority_preemption.adoc

```diff
@@ -88,51 +87,77 @@ spec:
   - name: nginx
     image: nginx
     imagePullPolicy: IfNotPresent
-  priorityClassName: high-priority
+  priorityClassName: high-priority <1>
 ----
 
-=== Effect of Pod priority on scheduling order
+<1> Speciffy the priority class to use with this pod. 
 
-When Pod priority is enabled, scheduler orders pending Pods by their priority and a pending Pod is placed ahead of other pending Pods with lower priority in the scheduling queue. As a result, the higher priority Pod may by scheduled sooner that Pods with lower priority if its scheduling requirements are met. If such Pod cannot be scheduled, scheduler will continue and tries to schedule other lower priority Pods.
```

Comment:
Why remove this section?

Reply:
@ravisantoshgudimetla I moved the paragraph higher in the topic. See your comment above ^^

Reply:
Ohh I missed it, thanks.